vagrant/provisioning/hypernode.sh

```diff
@@ -146,19 +146,50 @@ if ! $varnish_enabled; then
     varnishadm vcl.use nocache
 fi
 
-# ufw is disabled by default with an upstart override in the boxfile image because sometimes 
-# the firewall gets in the way when mounting the directories with specific synced folder fs types
+# ufw is disabled by default in the boxfile image because sometimes the firewall gets 
+# in the way when mounting the directories with specific synced folder fs types
 if $firewall_enabled; then
```

Comment:
yeah but that was never scripted for precise either, and that's only for the use case when people manually run `vagrant provision` again on an already existing instance after changing the firewall setting in the local.yml, and I think this script isn't completely idempotent in other places as well. I'd rather rewrite all this post-provisioning stuff in ansible some day instead of crufting all this stuff into this bash script